index.html

```diff
@@ -291,6 +300,20 @@ <h3 id="rec-track"> Normative Specifications </h3>
               Group before being considered in the Working Group.
             </p>
           </dd>
+          <dt> <a href="https://webscreens.github.io/openscreenprotocol/">Open Screen Protocol</a> </dt>
+          <dd>
+            <p>
+              A suite of network protocols that allow user agents to implement
```

Comment:
Does the charter clearly state that the protocol suite described in the Scope section, will be the Open Screen Protocol?

Reply:
It does not, but it does not really state either that the Presentation API and Remote Playback API match the description of the APIs in the Scope section. That does not strike me as odd, but we can be more explicit if you feel that's better.

Reply:
It seems fine as is then!